engines/scumm/script_v5.cpp

```diff
@@ -2490,7 +2490,7 @@ void ScummEngine_v5::walkActorToActor(int actor, int toActor, int dist) {
 	if (_game.version <= 2) {
 		x /= V12_X_MULTIPLIER;
 		y /= V12_Y_MULTIPLIER;
-	}
+	} 
```

Comment:
This whole file needs to be reverted, as it contains only the whitespace changes.